spec/models/queries/projects/factory_spec.rb

```diff
@@ -869,7 +869,7 @@
 
     it "is ordered by lft asc" do
       expect(find.orders.map { |order| [order.attribute, order.direction] })
-        .to eq([["lft", :asc]])
+        .to eq([%i[lft asc]])
     end
```

Comment:
The changes done around duplicating an existing query are not reflected in these specs.

Reply:
Indeed, it is already checked in feature tests, but should be in factory spec too, so added in de51c5a63e3a6ef79027b535427a193a31427e06, also some fixed for existing tests in 2cc9f9cc46dbb996b95471c7c4202754417ca407 and b70fb78997802ae03f7418898b69dfd421a36050